test/e2e_env/kubernetes/meshhealthcheck/api.go

```diff
@@ -0,0 +1,118 @@
+package meshhealthcheck
+
+import (
+	"fmt"
+
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/kumahq/kuma/test/e2e_env/kubernetes/env"
+	. "github.com/kumahq/kuma/test/framework"
+)
+
+func API() {
+	meshName := "meshhealthcheck-api"
+
+	BeforeAll(func() {
+		err := NewClusterSetup().
+			Install(MeshKubernetes(meshName)).
+			Setup(env.Cluster)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	E2EAfterEach(func() {
+		Expect(
+			k8s.RunKubectlE(env.Cluster.GetTesting(), env.Cluster.GetKubectlOptions(), "delete", "meshhealthchecks", "-A", "--all"),
```

Comment:
please update it to use `DeleteMeshResources(` (fetch master for it)

Reply:
I am working on resolving conflicts and removing duplicated methods